### ISSUE TITLE
Question: Meaning of "Number" for groups, logbooks and tags

### ISSUE THREAD

**Issue by eddybl** [wanglin86769/clog2#3]
As I am currently exploring CLog, I am trying to understand the meaning of the "Number" column which shows up in the group management, logbook management and tag management pages.

Is this used for sorting?

**Comment by wanglin86769** [comment]
Yes, the `Number` column is used for sorting and the items with small numbers come first.

If the `Number` column is empty, the items will be displayed in the order they are added to database.
